lighthouse-core/test/audits/byte-efficiency/unused-javascript-test.js

```diff
@@ -56,134 +56,107 @@ function generateUsage(url, ranges, transferSize = 1000) {
     };
   });
 
-  return {url, functions, networkRecord: {transferSize}};
+  return {url, functions};
 }
 
-describe('UnusedJavaScript audit', () => {
-  describe('#computeWaste', () => {
-    it('should identify used', () => {
-      const usage = generateUsage('myscript.js', [[0, 100, true]]);
-      const result = UnusedJavaScript.computeWaste(usage);
-      assert.equal(result.unusedLength, 0);
-      assert.equal(result.contentLength, 100);
-    });
-
-    it('should identify unused', () => {
-      const usage = generateUsage('myscript.js', [[0, 100, false]]);
-      const result = UnusedJavaScript.computeWaste(usage);
-      assert.equal(result.unusedLength, 100);
-      assert.equal(result.contentLength, 100);
-    });
-
-    it('should identify nested unused', () => {
-      const usage = generateUsage('myscript.js', [
-        [0, 100, true], // 40% used overall
-
-        [0, 10, true],
-        [0, 40, true],
-        [20, 40, false],
-
-        [60, 100, false],
-        [70, 80, false],
-
-        [100, 150, false],
-        [180, 200, false],
-        [100, 200, true], // 30% used overall
-      ]);
+function makeJsUsage(...usages) {
+  return usages.reduce((acc, cur) => {
+    acc[cur.url] = acc[cur.url] || [];
+    acc[cur.url].push(cur);
+    return acc;
+  }, {});
+}
 
-      const result = UnusedJavaScript.computeWaste(usage);
-      assert.equal(result.unusedLength, 130);
-      assert.equal(result.contentLength, 200);
-    });
+describe('UnusedJavaScript audit', () => {
+  const domain = 'https://www.google.com';
+  const scriptUnknown = generateUsage(domain, [[0, 3000, false]]);
+  const scriptA = generateUsage(`${domain}/scriptA.js`, [[0, 100, true]]);
+  const scriptB = generateUsage(`${domain}/scriptB.js`, [[0, 20000, true], [0, 5000, false]]);
```

Comment:
did thresholds change if this number was increased so much?

Reply:
I accidentally had the threshold checking the wasted bytes response size. Fixed it - these test values are now unchanged.